src/lib/__snapshots__/scrape.test.ts.snap

```diff
@@ -0,0 +1,825 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`al-pacino 1`] = `
+Object {
+  "author": "Tom Kershaw",
+  "content": Array [
+    Object {
+      "sourceTitle": undefined,
+      "sourceUrl": undefined,
+      "text": "Al Pacino was born in Manhattan, New York and grew up in The Bronx, New York.",
+      "type": "sentence",
+    },
+    Object {
+      "sourceTitle": undefined,
+      "sourceUrl": undefined,
+      "text": "Pacino is the son of full-on Italians and was raised as such, which includes Catholicism. But it seems to have been only in name that Pacino was Catholic. He said:",
+      "type": "sentence",
+    },
+    Object {
+      "sourceTitle": "Al Pacino's Remembrance of Things (and Flings) Past",
+      "sourceUrl": "http://www.moviecrazed.com/outpast/alpacdowney.html",
+      "text": "My parents were Italian, and they had me baptized, but I was not raised in the Catholic tradition. I come from a broken home.",
+      "type": "quote",
+    },
+    Object {
+      "sourceTitle": "Al Pacino's Remembrance of Things (and Flings) Past",
+      "sourceUrl": "http://www.moviecrazed.com/outpast/alpacdowney.html",
+      "text": "Pacino is probably referring to the fact that his parents divorced when he was two-years-old and, contrary to Catholic tradition, his father was married four times.",
+      "type": "sentence",
+    },
+    Object {
+      "sourceTitle": "Al Pacino's Biography",
+      "sourceUrl": "http://www.oceleb.com/al-pacino/biography.html",
+      "text": "So, Pacino indicates that he's not really a Catholic. Then what is he? This is where things get difficult. Much like his fellow iconic, mobster movie stars Marlon Brando and Robert De Niro, Pacino won't talk about it in interviews. Not talking about his religion might be a precondition to giving an interview. Some sources cite him as an agnostic,",
+      "type": "sentence",
+    },
+    Object {
+      "sourceTitle": undefined,
+      "sourceUrl": undefined,
+      "text": "which makes sense. It's the easy way out of religious commitment. He did once give his philosophy of mankind–which is cryptic and almost meaningless:",
+      "type": "sentence",
+    },
```

Comment:
Regarding the two objects above: the `text` field of the first object ends in a `,` and the `text` field of the object below it starts with the lower-case word `which`. Do these objects contain sentences? If they do, how will we know when a paragraph starts and ends? If they contain paragraphs, then why do we have one paragraph that ends in a `,`?

Reply:
The scraper works at the text node level. Because the first object has a source (`"Some sources cite him as an agnostic"`), the text nodes are separated with a `<sup>` element that contains a link to a reference in the reference list at the footer of the page.

I think this makes sense, because the reference is intended for the first sentence only. When we show this content on the new app, we will need to recreate these objects as if they were text nodes.

I think the scraper should only worry about extracting the data, not formatting it in a meaningful way.